js/index.js

```diff
@@ -1,491 +0,0 @@
-    stroke(211,181,9)
-    rect(insideX, height - 600 + 42.5, insideW, 300);
-    
-
-    // fill(254,204,0);
-    // strokeWeight(3)
-    // stroke(211,181,9)
-    // quad(transporterX - 250  , height - 600,transporterX, height - 600 + 30, transporterX , height - 600 + 355, transporterX - 250 , height - 600 + 355 - 40)
-
-    // fill(254,204,0);
-    // strokeWeight(3)
-    // stroke(211,181,9)
-    // quad(transporterW + transporterX + 250, height - 600 + 355 - 40,transporterX + transporterW,height - 600 + 355, transporterX + transporterW, height - 600 + 30 , transporterW + transporterX + 250 , height - 600);
-    fill(254,204,0);
-    strokeWeight(3)
-    stroke(211,181,9)
-    quad(windowWidth - transporterW + 300, height - 600 - 30,transporterX, height - 600, transporterX + transporterW, height - 600, windowWidth , height - 600-30);
-
-    fill(254,204,0);
-    strokeWeight(3)
-    stroke(211,181,9)
-    quad(windowWidth, 300 + height - 600,transporterX + transporterW,height - 600 + 355 +25, transporterX + transporterW, height - 600, windowWidth , height - 600-30);
-
-
-    //belt
-    stroke(0)
-    fill(30);
-    rect(0, height - 140, width, 125);
-
-      
-    fill(150);
-    strokeWeight(1)
-    rect(0, height-150, width, 125);
-
-    let beltWidth = 100; 
-    let beltHeight = 125; 
-    let numBelts = Math.round(width / beltWidth); 
-    for (let i = 0; i < numBelts; i++) {
-      strokeWeight(1)
-      rect(beltX + (i * beltWidth), height - 150, beltWidth, beltHeight);
-      fill(150);
-    }
-
-  //score
-  fill(250);
-  textSize(30);
-  text(`score: ${score}`, 100,50); 
-
-    //draw packages
-    randomARR.forEach((package)=>{
-      package.y = height - package.h - 75
-
-      noStroke()
-      fill(184,136,91)
-      rect(package.x, package.y, package.w, package.h) 
-      //tape
-      stroke(161,115,76);
-      strokeWeight(10);
-      drawTape(package.x + 5,package.y+5,package.w - 10,package.h-10)
-     
-      //sticker 
-      stroke(229,226,228)
-      fill(229,226,228)
-      rect(package.x + 15,package.y + 20, 30, 15)
-
-      //text in sticker
-      strokeWeight(0)
-      fill(0);
-      textSize(12);
-      text(package.text, package.x + 30, package.y + 30);
-
-
-      //make package move
-      package.x += package.speed
-      package.secX += package.speed
-      
-      //make package stop
-
-      if (package.x + package.w > width - 20) {
-      package.speed = 0;
-      }
-
-      for (let i = 0; i < randomARR.length; i++) {
-        for (let j = i + 1; j < randomARR.length; j++) {
-          const packageA = randomARR[i];
-          const packageB = randomARR[j];
-          if(packageA.speed === 0){  
-            if (packagesOverlap(packageA, packageB)) {
-            packageA.speed = 0;
-            packageB.speed = 0;
-          }
-        }
-
-      
-      }
-    }
-
-    //set game over condition
-   if(package.speed === 0 && package.x <= 20 ){
-    gameOver();
-    console.log("game over")
-   }
```

Comment:
Clever way of checking failure condition and clever way of making packages stack up in general, well done